### PR TITLE
[@types/chart.js] Added missing axis field to hover options

### PR DESCRIPTION
**Pull request by tonybadguy**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://www.chartjs.org/docs/latest/general/interactions/>>

### CODE REVIEW

**Comment by lekoaf** [comment]
How about this instead?

axis?: ’x’ | ’y’ | ’xy’

Since those are the only options the user have.

File: types/chart.js/index.d.ts [hunk 1]
```diff
@@ -442,6 +443,7 @@ declare namespace Chart {
         mode?: InteractionMode;
         animationDuration?: number;
         intersect?: boolean;
+        axis?: string;
```
Comment (lekoaf) [inline types/chart.js/index.d.ts]: How about this instead?

axis?: ’x’ | ’y’ | ’xy’

Since those are the only options the user have.

**Review by lekoaf — CHANGES_REQUESTED** [review]

File: types/chart.js/index.d.ts (hunk 1 above)
Comment (lekoaf) [reply types/chart.js/index.d.ts]: (I’m typing on my phone, hence the crappy formatting)

Comment (tonybadguy) [reply types/chart.js/index.d.ts]: good idea. updated.

**Review by lekoaf — APPROVED** [review]
LGTM

**Comment by uniqueiniquity** [comment]
Congratulations on your first DefinitelyTyped contribution!
Thank you for being a part of the community!

**Pull request merged**